### PR TITLE
Real supervision

### PR DESCRIPTION
**Pull request by benlangfeld**
@bklang ?

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011 Ben Langfeld, Jay Phillips
+Copyright (c) 2016 Ben Langfeld, Jay Phillips
```
Comment (bklang) [inline LICENSE.txt]: 2011-2016

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # [develop](https://github.com/adhearsion/ruby_ami)
+  * Breaking change: Removed the deprecated `RubyAMI::Client` because it is no longer relevant.
```
Comment (bklang) [inline CHANGELOG.md]: Doesn't this conflict with the other PR?

Comment (benlangfeld) [inline CHANGELOG.md]: It's just that one is based on the other.

**Comment by bklang** [comment]
The title of this PR is "Real supervision" but I only see one relevant code change: `async.run`. Is that really all that is required to have it properly supervised?

**Comment by benlangfeld** [comment]
> Is that really all that is required to have it properly supervised?

Not for it to *be* supervised, but for it to fit into Celluloid supervision semantics. I added a documentation note to clarify. I also corrected an omission which was updated in the docs but not implemented; I submitted this PR in the midst of production downtime and forgot about it.

**Comment by bklang** [comment]
:+1:

**Pull request merged**